scenario_test_tools/scripts/example_move_base.py

```diff
@@ -12,7 +12,9 @@
 
     succeeded = MoveBaseResult()
 
-    move_base = ScriptableMoveBase('/move_base', MoveBaseAction, result_delay=5)
+    result_delay = rospy.get_param("result_delay", 5)  # By default, wait 5 secs for the robot to arrive
+
+    move_base = ScriptableMoveBase('/move_base', MoveBaseAction, result_delay=result_delay)
```

Comment:
This also needs to be configurable. And please have it in the private namespace (`~`)

Reply:
sorry read @fmessmer 's comment too late (but the private namespace thing still needs to be done :) )

Reply:
Remapping didn't work? wtf?

Reply:
as the node needs to be in the namespace `corridor_shelf_aligning`, but remapping does that in the global namespace :point_right: so I would have to remap all topics

I think the renaming-of-the-node approach is more handy there

Reply:
Goal of this example is to be a good example rather than being exactly what we need in our tests, so I think even the private namespace is not according to that goal.